packages/integration-tests/src/test-e2e/sandbox/sandbox.test.template.ts

```diff
@@ -0,0 +1,107 @@
+import { after, before, describe, it } from 'node:test';
+import {
+  createTestDirectory,
+  deleteTestDirectory,
+  rootTestDir,
+} from '../../setup_test_directory.js';
+import { TestProjectCreator } from '../../test-project-setup/test_project_creator.js';
+import { TestProjectBase } from '../../test-project-setup/test_project_base.js';
+import { userInfo } from 'os';
+import { ampxCli } from '../../process-controller/process_controller.js';
+import {
+  ensureDeploymentTimeLessThan,
+  interruptSandbox,
+  rejectCleanupSandbox,
+  replaceFiles,
+  waitForConfigUpdateAfterDeployment,
+} from '../../process-controller/predicated_action_macros.js';
+import { BackendIdentifier } from '@aws-amplify/plugin-types';
+import { testConcurrencyLevel } from '../test_concurrency.js';
+import {
+  amplifySharedSecretNameKey,
+  createAmplifySharedSecretName,
+} from '../../shared_secret.js';
+
+/**
+ * Defines sandbox test
+ */
+export const defineSandboxTest = (testProjectCreator: TestProjectCreator) => {
+  void describe('sandbox test', { concurrency: testConcurrencyLevel }, () => {
```

Comment:
This has been taken from:
https://github.com/aws-amplify/amplify-backend/blob/5f46d8dc1f76d567065b2aa724ec571592bbfa1f/packages/integration-tests/src/test-e2e/sandbox.test.ts#L34-L109